src/ipa/rpi/vc4/data/ov64a40.json

```diff
@@ -0,0 +1,427 @@
+                "reference_Y": 9022
+            }
+        },
+        {
+            "rpi.noise":
+            {
+                "reference_constant": 0,
+                "reference_slope": 2.984
+            }
+        },
+        {
+            "rpi.geq":
+            {
+                "offset": 215,
+                "slope": 0.01121
+            }
+        },
+        {
+            "rpi.sdn": { }
+        },
+        {
+            "rpi.awb":
+            {
+                "priors": [
+                    {
+                        "lux": 0,
+                        "prior":
+                        [
+                            2000, 1.0,
+                            3000, 0.0,
+                            13000, 0.0
+                        ]
+                    },
+                    {
+                        "lux": 800,
+                        "prior":
+                        [
+                            2000, 0.0,
+                            6000, 2.0,
+                            13000, 2.0
+                        ]
+                    },
+                    {
+                        "lux": 1500,
+                        "prior":
+                        [
+                            2000, 0.0,
+                            4000, 1.0,
+                            6000, 6.0,
+                            6500, 7.0,
+                            7000, 1.0,
+                            13000, 1.0
+                        ]
+                    }
+                ],
+                "modes":
+                {
+                    "auto":
+                    {
+                        "lo": 2500,
+                        "hi": 8000
+                    },
+                    "incandescent":
+                    {
+                        "lo": 2500,
+                        "hi": 3000
+                    },
+                    "tungsten":
+                    {
+                        "lo": 3000,
+                        "hi": 3500
+                    },
+                    "fluorescent":
+                    {
+                        "lo": 4000,
+                        "hi": 4700
+                    },
+                    "indoor":
+                    {
+                        "lo": 3000,
+                        "hi": 5000
+                    },
+                    "daylight":
+                    {
+                        "lo": 5500,
+                        "hi": 6500
+                    },
+                    "cloudy":
+                    {
+                        "lo": 7000,
+                        "hi": 8600
+                    }
+                },
+                "bayes": 1,
+                "ct_curve":
+                [
+                    2300.0, 1.0522, 0.4091,
+                    2700.0, 0.7884, 0.4327,
+                    3000.0, 0.7597, 0.4421,
+                    4000.0, 0.5972, 0.5404,
+                    4150.0, 0.5598, 0.5779,
+                    6500.0, 0.4388, 0.7582
```

Comment:
This isn't a particularly high value, so I wonder if you might want to delete the "cloudy" AWB mode... I can't think it would do anything useful?

Reply:
Yes, currently our light source can only go up to 6500k, the color temperature range for cloud mode is 7000k to 8000k, I think we can indeed remove that, do you have any suggestions for the calibration process? Especially when there are not many light sources.

Reply:
Yes, it's probably best to remove it. On the other hand, if you've tried it and it actually does something useful, then it's fine to keep it. Getting hold of very blue lamps is tricky, I suppose you could buy some filters but these are often not great either. But I think leaving the top at 6500 is perfectly fine too.

Reply:
> Yes, it's probably best to remove it. On the other hand, if you've tried it and it actually does something useful, then it's fine to keep it. Getting hold of very blue lamps is tricky, I suppose you could buy some filters but these are often not great either. But I think leaving the top at 6500 is perfectly fine too.

Got it, thanks for your suggestion.